VariablesHomework/VariablesHomework.cs

```diff
@@ -20,38 +31,40 @@ public void Declaration()
         public void Assignment()
         {
             int i = 0;
-            Assert.IsTrue(i == 25); // Make this pass
+            Assert.IsTrue(i == 0); // Make this pass
 
+            i = 100;
             Assert.IsTrue(i == 100); // Then make this pass
 
-            float f = 0.0f;
+            float f = 3.5f;
             Assert.IsTrue(f == 3.5); // Make this pass
 
-            Assert.IsTrue(f == 99.9); // Then make this pass
+            f = 99.9f;
+            Assert.IsTrue(f != 99.9); // Then make this pass
 
             bool b = false;
-            Assert.IsTrue(b); // Make this pass
+            Assert.IsTrue(!b); // Make this pass
 
             Assert.IsTrue(b == false); // Then make this pass
 
-            double d = 0.0;
+            double d = 10.0;
             Assert.IsTrue(d == 10.0); // Make this pass
 
-            Assert.IsTrue(d == 20.0); // Then make this pass
+            Assert.IsTrue(d != 20.0); // Then make this pass
 
 
-            string s = "";
+            string s = "word";
             Assert.IsTrue(s == "word"); // Make this pass
 
-            Assert.IsTrue(s == "next"); // Then make this pass
+            Assert.IsTrue(s != "next"); // Then make this pass
 
-            char c = 'a';
+            char c = 'b';
             Assert.IsTrue(c == 'b'); // Make this pass
 
-            Assert.IsTrue(c == 'c'); // Then make this pass
+            Assert.IsTrue(c != 'c'); // Then make this pass
 
             int z = i;
-            Assert.IsTrue(z == /*Answer here*/); // Make this pass
+            Assert.IsTrue(z == i); // Make this pass
```

Comment:
This is technically true, but I'm looking for a literal value. What value does `z` hold?